module-apps/apps-common/AppSwitchReason.hpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+namespace app
+{
+    enum class SwitchReason
+    {
+        SwitchRequest,
+        PhoneLock,
+        Popup
```

Comment:
probably functionality of `PhoneLock` and `Popup` switch reason could be merged

Reply:
probably. but this pr doesn't have in goal rewriting whole popup related handling codebase